content/tutorials/_live-flight-tracking.textile

```diff
@@ -0,0 +1,27 @@
+---
+title: Live flight tracking app
+excerpt: Build a live flight tracker in React Native using an open data stream from Ably Hub
+section: tutorials
+category:
```

Comment:
not sure how this got missed in the sheet but this needs to have a 'channels' tag as well

Reply:
done